versioned_docs/version-2.0/batch-processing/quiz.md

```diff
@@ -50,6 +50,7 @@ In standard SQL, it aggregates rows that share the same grouping key into a sing
 
 
 <details><summary>What’s one scenario where using Window functions are advantageous over GroupBy aggregations?</summary>
+
```

Comment:
This and the following white space are all that are needed to make the links work. 😸